src/components/policy/src/policy/include/policy/policy_manager_impl.h

```diff
@@ -187,6 +189,24 @@ class PolicyManagerImpl : public PolicyManager {
 
     virtual std::string RetrieveCertificate() const OVERRIDE;
 
+    class RetrySequenceTimewatch {
```

Comment:
@AGritsevich for what do you need this watcher? Why don't you just use common Timer?